R/makeExtractFDAFeatMethodBsignal.R

```diff
@@ -0,0 +1,35 @@
+#' @title Bspline mlq features
+#'
+#' @description
+#' The function extracts features from functional data based on the Bspline fit.
+#' For more details refer to \code{\link[FDboost]{bsignal}}.
+#'
+#' @param bsignal.knots [\code{integer}]\cr
+#' The number of knots for bspline.
+#' @param bsignal.df [\code{numeric}]\cr
+#' The effective degree of freedom of penalized bspline.
+#' @return [\code{data.frame}].
+#' @export
+#' @family fda_featextractor
+extractFDABsignal = function(bsignal.knots = 10L, bsignal.df = 3) {
```

Comment:
capital B or lowercase b for bsignal?

Reply:
well, bsignal is the orginal version but here use Camel case, so it should be upper case. how do you think?

Reply:
solved